index.html

```diff
@@ -113,6 +118,14 @@ <h2>Security and Privacy Considerations</h2>
       <a data-cite="URL#concept-url-scheme">scheme</a>.
     </p>
 
+    <p>
+      A <dfn>cookie name filter</dfn> is the name of a cookie that will be used by
```

Comment:
I initially misread this sentence because it sounded like the user agent would invalidate the data by means of the cookie (i.e., editing the cookie somehow). But actually the cookie change just signals the user agent to invalidate using its own mechanism.

I'd suggest something like: "A cookie name filter is the name of a cookie that signals the user agent to invalidate the contents of the feed..."